components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -155,6 +153,12 @@ spec:
             
           - name: TYK_DB_HTTPSERVEROPTIONS_USESSL
             value: "{{ .Values.global.tls.dashboard }}"
+          - name: TYK_DB_HTTPSERVEROPTIONS_MINVERSION
+            value: "771"
```

Comment:
just asking :) why do we need to specify min tls version?  isn't optional field?

Reply:
afaik, tls < 1.2 is deprecated so it seems like a good thing to enforce on new charts

Reply:
Yeah this was added to stay consistent with the gateway.